decidim-admin/app/views/decidim/admin/organization/_form.html.erb

```diff
@@ -25,3 +25,7 @@
 <div class="field">
   <%= form.file_field :logo %>
 </div>
+
+<div class="field">
+  <%= form.check_box :show_statistics, label: t(".questions.show_statistics") %>
```

Comment:
You might not need the `label` part here. If you need it, then add the missing locale